### PR TITLE
track native token balance changes from evm txs

### PR DESCRIPTION
**Pull request by Andrew7234**
https://app.clickup.com/t/866a11n9q

Todo:
- Testing this against cobalt

### CODE REVIEW

**Review by aefhm — APPROVED** [review]
LG caveat my context may be limited

File: analyzer/runtime/runtime.go [hunk 1]
```diff
@@ -318,6 +319,23 @@ func (m *Main) queueDbUpdates(batch *storage.QueryBatch, data *BlockData) {
...
 		)
+
+		// Handle native-token transfers from transactions.
+		// These transfers do not emit events and are not covered
+		// by the accounts module, so we process them here.
+		if transactionData.Method == "evm.Call" {
+			body := transactionData.Body.(*evm.Call)
+			if len(body.Data) == 0 {
+				amount := common.QuantityFromBytes(body.Value)
+				batch.Queue(
```
Comment (pro-wh) [inline analyzer/runtime/runtime.go:330]: can we do that here https://github.com/oasisprotocol/oasis-indexer/blob/316cd3b3bf5eaa217e026d33c9a5086bb98307d5/analyzer/runtime/extract.go#L382

Comment (mitjat) [reply analyzer/runtime/runtime.go:330]: +1, the runtime analyzer is more structured than the consensus analyzer, and tends to first store all the analysis results into a big struct, then convert that struct into sql statements in a separate step.

Comment (Andrew7234) [reply analyzer/runtime/runtime.go]: updated!

File: analyzer/runtime/runtime.go [hunk 2]
```diff
@@ -318,6 +318,16 @@ func (m *Main) queueDbUpdates(batch *storage.QueryBatch, data *BlockData) {
 			error_code,
 			error_message,
 		)
+
+		if transactionData.IsNativeTransfer {
+			batch.Queue(
+				queries.RuntimeNativeBalanceUpdate,
```
Comment (pro-wh) [inline analyzer/runtime/runtime.go:324]: do we ever subtract from the sender account?

**Comment by Andrew7234** [comment]
Closing this since we decided to track native balances differently.

**Pull request closed without merging**